lib/sablon/parser/mail_merge.rb

```diff
@@ -2,33 +2,34 @@ module Sablon
   module Parser
     class MailMerge
       class MergeField
-        KEY_PATTERN = /^\s*MERGEFIELD\s+([^ ]+)\s+\\\*\s+MERGEFORMAT\s*$/
+        KEY_PATTERN = /^\s*MERGEFIELD\s+(.+?)\s+\\\*\s+MERGEFORMAT\s*$/
```

Comment:
Previous expression captured exactly one word without spaces in it. The new if-statement format uses spaces, so instead we are now capturing everything between `MERGEFIELD` and the first `\*`.